azure.yaml

```diff
@@ -3,9 +3,63 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v11.3.0
+  name: v11.3.1
 spec:
   state: active
+  date: 2020-05-12T11:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.1
+    - name: chart-operator
+      version: 0.13.0
+    - name: coredns
+      componentVersion: 1.6.5
+      version: 1.1.8
+    - name: external-dns
+      componentVersion: 0.5.18
+      version: 1.2.0
+    - name: kube-state-metrics
+      componentVersion: 1.9.5
+      version: 1.0.5
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - name: net-exporter
+      version: 1.7.0
+    - name: nginx-ingress-controller
+      componentVersion: 0.30.0
+      version: 1.6.9
```

Comment:
Please hold with this one, 0.23.9 cluster-operator seems to expect that nginx 1.6.10 is being used, @whites11 correct?
Extra context see: https://github.com/giantswarm/cluster-operator/pull/1050

Reply:
I think this PR can be closed, I've added this change to your PR Stevo: https://github.com/giantswarm/releases/pull/269 
it's also renamed from 11.3.1 to 11.4.0.

Reply:
Cool, thanks for heads up. Will close this one then. @tomahawk28 feel free to reopen if you think differently